vitest.config.ts

```diff
@@ -8,6 +8,10 @@ export default mergeConfig(
     test: {
       setupFiles: ['src/installSesLockdown.ts'],
       environment: 'happy-dom',
+      exclude: [
+        ...configDefaults.exclude,
+        'tests/e2e/**'
```

Comment:
alternately we could consider `tests` to be the purview of vitest and have synpress run in `/integration-tests` or `/e2e-tests`. I advise this to simplify the configurations but I defer to whatever @samsiegart prefers.

Reply:
if our goal is to simply config then just a head up: changing this path will require updating the `synpress.config.js` file since it uses `tests/e2e/**` as its default path

Reply:
Thanks for the heads up, I think we can consider separating the directories but not a blocker